src/Donations/Endpoints/ListDonations.php

```diff
@@ -126,6 +127,16 @@ public function registerRoute()
                             'columns',
                         ],
                     ],
+                    'postStatus' => [
+                        'type' => 'string',
+                        'required' => false,
+                        'default' => 'active',
+                        'enum' => [
+                            'active',
+                            'trash',
+                        ],
+                        'description' => 'Filter donations by WordPress post status: active (all except trash), or trash.'
```

Comment:
Again here let's keep the context of the donation - not WordPress posts (which is the storage we use under the hood and will change in the future)

```suggestion
                        'description' => 'Filter donations by Donation status: active (all except trash), or trash.'
```

Reply:
Resolved: 4b24aa62c811897cdaa5127f1859cdd294d0c5ce